app/views/admin/adventure_mission_groups/edit.html.erb

```diff
@@ -1,20 +1,25 @@
 <div class="col-md-12">
-  <h1><%= t('.title') %></h1>
-   <ul class="list-group">
-      <li class="list-group-item">
-          <h3><%= @adventure_mission_group.adventure_mission.title %> </h3>
-        </li>
-        <li class="list-group-item">
-          <span class="badge"><%= @adventure_mission_group.adventure_mission.max_points %></span>
-          <%= AdventureMission.human_attribute_name(:max_points) %>
-        </li>
-        <li class="list-group-item">
-          <%= simple_form_for([:admin, @adventure_mission_group]) do |f| %>
-            <%= f.input :points %>
-            <%= f.input :adventure_mission_id, as: :hidden, input_html: { value: @adventure_mission_group.adventure_mission.id }%>
-            <%= f.button :submit, value: t('.save') %>
-          <% end %>
-        </li>
-      </ul>
-    <%= link_to t('.destroy'), admin_adventure_mission_group_path(@adventure_mission_group), method: :delete, data: { confirm: t('.confirm_destroy') } , class: 'btn danger' %>
+  <div class="headline">
+    <h1><%= "#{title(t('.title'))}" %> </h1>
+  </div>
+
+  <ul class="list-group">
+    <li class="list-group-item">
+        <h3><span><%= @group.name %></span>: <%= @adventure_mission_group.adventure_mission.title %></h3>
+    </li>
+    <li class="list-group-item">
+      <span class="badge"><%= @adventure_mission_group.adventure_mission.max_points %></span>
+      <%= AdventureMission.human_attribute_name(:max_points) %>
+    </li>
+    <li class="list-group-item">
+      <%= simple_form_for([:admin_adventure, @adventure_mission_group]) do |f| %>
+        <%= f.input :points %>
+        <%= f.input :pending, label: t('.pending'), as: :radio_buttons %>
+        <%= f.input :adventure_mission_id, as: :hidden, input_html: { value: @adventure_mission_group.adventure_mission.id }%>
+        <%= f.button :submit, value: t('.save') %>
```

Comment:
Only show the `pending` option if acceptance is required. This results in you needing to handle if `pending` is `nil` in your controller. See comment in the corresponding controller.

Also, I think that it is better to not set a `value` on the submit button.
```suggestion
        <% if @adventure_mission_group.adventure_mission.require_acceptance %>
          <%= f.input :pending, label: t('.pending'), as: :radio_buttons %>
        <% end %>
        <%= f.input :adventure_mission_id, as: :hidden, input_html: { value: @adventure_mission_group.adventure_mission.id } %>
        <%= f.button :submit %>
```

Reply:
Submit value I believe was there before